src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/checkBalance.js

```diff
@@ -27,9 +29,14 @@ function getFormattedProperties(checkBalanceResponse, orderAmount) {
   return {};
 }
 
+// eslint-disable-next-line complexity
```

Comment:
would be better if the function can be simplified instead of disabling the complexity rule